### PR TITLE
snap refactor 1: remove v1

### PR DESCRIPTION
**Pull request by choleraehyq**
@siddontang @hhkbp2 @zhangjinpeng1987 PTAL

### CODE REVIEW

File: src/raftstore/store/snap.rs [hunk 1]
```diff
@@ -988,26 +471,18 @@ mod v2 {
                 return Ok(());
             }
             for cf_file in &mut self.cf_files {
-                if plain_file_used(cf_file.cf) {
```
Comment (siddontang) [inline src/raftstore/store/snap.rs]: is it proper to use SST for lock CF? @zhangjinpeng1987

Comment (zhangjinpeng1987) [inline src/raftstore/store/snap.rs]: It is too expensive for lock cf to use sst file.

Comment (choleraehyq) [inline src/raftstore/store/snap.rs]: I totally remove v1 related code in this PR to kill clear. CF_LOCK will return to use plain file in another PR.

Comment (siddontang) [inline src/raftstore/store/snap.rs]: 😓  Please add a comment here.

**Comment by siddontang** [comment]
Rest LGTM

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**